web-components/src/components/code-editor/scss/settings.scss

```diff
@@ -0,0 +1,10 @@
+$md-code-editor-height: rem-calc(300);
+$md-code-editor-indent: rem-calc(10);
+$md-code-editor-color: $md-white-100;
+$md-code-editor-bg: $md-blue-95;
+$md-code-editor-header-bg: $md-blue-90;
```

Comment:
I noticed all the color definitions are hard coded and that there aren't light/dark colors defined specifically for this component. Is that intentional?

Reply:
We have just one color theme for code editor